tests/bsim/bluetooth/tests.nrf5340bsim_nrf5340_cpuapp.txt

```diff
@@ -2,5 +2,5 @@
 # This file is used in CI to select which tests are run
 tests/bsim/bluetooth/ll/conn/tests_scripts/basic_conn_encrypted_split_privacy.sh
 tests/bsim/bluetooth/ll/bis/tests_scripts/broadcast_iso.sh
-tests/bsim/bluetooth/samples/
+tests/bsim/bluetooth/samples/central_hr_peripheral_hr/
```

Comment:
battery_service bsim is executed as part of nrf52 bsim
This is to avoid battery_service bsim test  (which is also under bluetooth/samples) to get executed under nr5340 bsim